src/site/markdown/server/manage-webadmin.md

```diff
@@ -2041,6 +2042,198 @@ curl -XDELETE http://ip:port/sieve/quota/users/user@domain.com
 Response codes:
  - 204: Operation succeeded
 
+
+## ReIndexing
+
+ - [ReIndexing all mails](#ReIndexing_all_mails)
+ - [ReIndexing a user mails](#ReIndexing_a_user_mails)
+ - [ReIndexing a mailbox mails](#ReIndexing_a_mailbox_mails)
+ - [ReIndexing a single mail](#ReIndexing_a_single_mail)
+
+Be also aware of the limits of these APIs:
+
+Warning: During the re-indexing, the result of search operations might be altered.
+
+Warning: Canceling this task should be considered unsafe as it will leave the currently reIndexed mailbox as partially indexed.
+
+Warning: While we have been trying to reduce the inconsistency window to a maximum (by keeping track of ongoing events),
+concurrent changes done during the reIndexing might be ignored.
+
+### ReIndexing all mails
+
+```
+curl -XPOST http://ip:port/index?task=reIndex
+```
+
+Will schedule a task for reIndexing all the mails stored on this James server.
+
+The response to that request will be the scheduled `taskId` :
+
+```
+{"taskId":"5641376-02ed-47bd-bcc7-76ff6262d92a"}
+```
+
+Positionned headers:
+
+ - Location header indicates the location of the resource associated with the scheduled task. Example:
+
+```
+Location: /tasks/3294a976-ce63-491e-bd52-1b6f465ed7a2
+```
+
+Response codes:
+
+ - 201: Success. Corresponding task id is returned.
+ - 400: Error in the request. Details can be found in the reported error.
```

Comment:
I saw in Swagger doc, there is one more response code: 500 documented.

Reply:
It is implicit in webadmin website doc: see the top of the file....